Mobile/starter_project_dark_knights/lib/features/Authentication/presentation/screen/log_in_screen.dart

```diff
@@ -0,0 +1,206 @@
+import 'package:flutter/material.dart';
+
+class Login extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final timeDisplay = Text("9:41");
+
+    final iconRow = Row(
+      mainAxisAlignment: MainAxisAlignment.end,
+      children: [
+        Transform.scale(scale: 0.8, child: Icon(Icons.signal_cellular_4_bar)),
+        SizedBox(width: 8),
+        Transform.scale(scale: 0.8, child: Icon(Icons.wifi)),
+        SizedBox(width: 8),
+        Transform.scale(scale: 0.8, child: Icon(Icons.battery_full)),
+      ],
+    );
+
+    return MaterialApp(
+      home: Scaffold(
+        body: Column(
+          children: [
+            Container(
+              padding: const EdgeInsets.symmetric(horizontal: 16, vertical: 8),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  timeDisplay,
+                  iconRow,
+                ],
+              ),
+            ),
+            Image.asset(
+              'images/a2sv.jpg',
+              //width: 200, // set the width of the image
+              height: 150, // set the height of the image
+            ),
+
+            //Container(height: 150, child: Image.asset('a2sv.jpg')),
+            Container(
+              width: double.infinity,
+              child: ClipRRect(
+                borderRadius: const BorderRadius.only(
+                  topLeft: Radius.circular(20.0),
+                  topRight: Radius.circular(20.0),
+                ),
+                child: Container(
+                  height: 50,
+                  color: Colors.blue[900],
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      Text(
+                        "LOGIN",
+                        style: TextStyle(

```

Comment:
Avoid giving styles like this. Define All styles inside the core/utils directory and make use of them whenever needed.